tests/test_worker.py

```diff
@@ -4,16 +4,27 @@
 import subprocess
 import urllib.request
 from unittest import mock
-from sanic.worker import GunicornWorker
-from sanic.app import Sanic
 import asyncio
-import logging
 import pytest
+from sanic.app import Sanic
+try:
+    from sanic.worker import GunicornWorker
+except ImportError:
+    pytestmark = pytest.mark.skip(
+        reason="GunicornWorker Not supported on this platform"
+    )
+    # this has to be defined or pytest will err on import
+    GunicornWorker = object
```

Comment:
We can use `@pytest.mark.skipif` instead.

Reply:
does skipif work on class definitions? The issue I had here was that GunicornTestWorker would fail to build if we didn't define GunicornWorker

Reply:
ah i see your point. Then you can use `collect_ignore.append("test_worker.py")` in conftest.py when detecting the platform is windows.